grafana-plugin/src/pages/schedules_NEW/Schedules.tsx

```diff
@@ -401,20 +367,10 @@ class SchedulesPage extends React.Component<SchedulesPageProps, SchedulesPageSta
   };
 
   handleSchedulesFiltersChange = (filters: SchedulesFiltersType) => {
-    this.setState({ filters }, this.debouncedUpdateSchedules);
-  };
-
-  applyFilters = () => {
-    const { filters } = this.state;
-    const { store } = this.props;
-    const { scheduleStore } = store;
-
-    // scheduleStore.updateItems(filters.searchTerm);
```

Comment:
@maskin25  should this be in here or not? `applyFilters` didn't achieve anything

Reply:
yes, leave it as is, please, this is a stub for https://github.com/grafana/oncall/issues/666

Reply:
[9f1ba50](https://github.com/grafana/oncall/pull/678/commits/9f1ba5060f15928230f4b3a52a7561cd51448c96) reverts this change to keep `applyFilters`